gfx/MShockXotto+/pngs_tiles_32x32/misc/highlight.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "highlight",
```

Comment:
Make one json file with an array  in the id field listing all the uses of cursor, including `cursor` right now with the json setup this way there is no sprite for cursor